deployments/data-hub/release-data-hub--stg.yaml

```diff
@@ -134,7 +134,7 @@ spec:
       - name: MATERIALIZE_BIGQUERY_VIEWS_SCHEDULE_INTERVAL
         value: "@hourly"
       - name: SCHEDULE_INTERVAL
```

Comment:
We probably should rename that variable, this is for the maintenance?

Reply:
Yes it is for it, we can change.